packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.Props.ts

```diff
@@ -30,6 +30,11 @@ export interface IProgressIndicatorProps {
   percentComplete?: number;
 
   /**
+   * Text alternative of progress status
```

Comment:
Yes, thats the scenario eg. we have an import workflow with multiple steps like "calculating objects ..." , "pushing repository" etc . I will update the comment to reflect the use case, thanks